src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -10,24 +10,35 @@ extern "C" {
 #include <sys/stat.h>
 #include <fcntl.h>
 
-#define FARF_LOW 1
 #include "HAP_farf.h"
 #include "HAP_power.h"
 
 }
 
 #include "../HalideRuntime.h"
+#include "log.h"
 
 typedef halide_hexagon_remote_handle_t handle_t;
 typedef halide_hexagon_remote_buffer buffer;
 
+Log global_log(1024 * 1024);
```

Comment:
Unconditionally allocating a megabyte for logging seems a bit excessive. I point it out because we might forget to turn this down later.